css/footer.css

```diff
@@ -0,0 +1,83 @@
+footer {
+  height: 846px;
+  position: relative;
+
+  /* text-align: center; */
+}
+
+.f-box-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
+  width: 801px;
+  height: 577px;
+  margin: auto;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.contact-me-text {
+  position: static;
+  width: 569px;
+  height: 102px;
+  left: 116px;
+  top: 68px;
+  font-family: Poppins, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.contact-h2 {
+  position: static;
+  width: 801px;
+  height: 56px;
+  left: 0;
+  top: 0;
+  text-align: center;
+}
+
+input {
+  height: 48px;
+  width: 447px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
```

Comment:
- [ ] Please use CSS classes for repeated variables like this color for example to keep your code clean, implement them in any place where you find repeated rules :)